modules/ocf_desktop/manifests/displaycontrol.pp

```diff
@@ -0,0 +1,6 @@
+class ocf_desktop::displaycontrol {
+
+  kmod::load { 'i2c_dev': }
+
+  package { [ 'i2c-tools', 'ddcutil' ]: }
```

Comment:
`i2c-tools` is a dep of `ddcutil`. We should only explicitly install the latter.